lib/tasks/misc.rake

```diff
@@ -0,0 +1,6 @@
+namespace :misc do
```

Comment:
maybe `mailer` is a better name? Just a nitpick - feel free to ignore

Reply:
I'd rather leave it as misc because this command will be rarely run. I like to keep my one-time runs in a misc namespace to denote the lack of importance.